scrapenews/spiders/rekordnorth.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+
+from scrapy.spiders import CrawlSpider, Rule
+from scrapy.linkextractors import LinkExtractor
+from scrapenews.items import ScrapenewsItem
+from datetime import datetime
+import pytz
+
+SAST = pytz.timezone('Africa/Johannesburg')
+
+class dfaSpider(CrawlSpider):
+    name = 'rekordnorth'
+    allowed_domains = ['rekordnorth.co.za']
+    start_urls = ['https://rekordnorth.co.za/category/news-headlines/local-news/']
+
+    link_extractor = LinkExtractor(
```

Comment:
try this - it seems to work fine - I was right - we should pass allowed_domains into LinkExtractor's allow_domains (note the different spelling :/ )

I noticed that the articles on this site all seem to have an integer as the first URL "part" - so it's easier to whitelist than blacklist.

```diff
diff --git a/scrapenews/spiders/rekordnorth.py b/scrapenews/spiders/rekordnorth.py
index b008475..8a86a81 100644
--- a/scrapenews/spiders/rekordnorth.py
+++ b/scrapenews/spiders/rekordnorth.py
@@ -14,70 +14,8 @@ class dfaSpider(CrawlSpider):
     start_urls = ['https://rekordnorth.co.za/category/news-headlines/local-news/']
 
     link_extractor = LinkExtractor(
-        allow=('https://rekordnorth.co.za', ),
-        deny=(
-            'rekordnorth.co.za/pretoria-north-news-team',
-            'rekordnorth.co.za/epapers',
-            'rekordnorth.co.za/feat/title-deeds/',
-            'rekordnorth.co.za/advertise-online/',
-            'rekordnorth.co.za/category/featured-content/',
-            'rekordnorth.co.za/beauty-health-all-year-round/',
-            'rekordnorth.co.za/rss-output/',
-            'rekordnorth.co.za/all4women',
-            'rekordnorth.co.za/national-news/',
-            'rekordnorth.co.za/local-motoring-news/',
-            'rekordnorth.co.za/category/image-gallery/',
-            'rekordnorth.co.za/category/image-gallery/general-news/',
-            'rekordnorth.co.za/category/image-gallery/community/',
-            'rekordnorth.co.za/category/image-gallery/sport-news/',
-            'rekordnorth.co.za/category/image-gallery/school-news/',
-            'rekordnorth.co.za/video-gallery/',
-            'rekordnorth.co.za/category/opinion-edition/letters/',
-            'rekordnorth.co.za/category/opinion-edition/blogs/',
-            'rekordnorth.co.za/category/opinion-edition/blogs/simply-delicious/',
-            'rekordnorth.co.za/category/lifestyle-news/competitions-in-pretoria-north/',
-            'rekordnorth.co.za/category/lifestyle-news/entertainment-news/',
-            'rekordnorth.co.za/events/event/',
-            'rekordnorth.co.za/category/news-headlines/',
-            'rekordnorth.co.za/category/sports-news/',
-            'rekordnorth.co.za/category/opinion-edition/',
-            'rekordnorth.co.za/category/lifestyle-news/',
-            'rekordnorth.co.za/online-classifieds/',
-            'rekordnorth.co.za/place-ad/',
-            'rekordnorth.co.za/terms-and-conditions/',
-            'rekordnorth.co.za/privacy-policy/',
-            'rekordnorth.co.za/builders/',
-            'rekordnorth.co.za/property/for-sale/',
-            'rekordnorth.co.za/property/to-rent/',
-            'rekordnorth.co.za/rest-assured/',
-            'rekordeast.co.za/parenting/',
-            'rekordnorth.co.za/support-local-business/',
-            'rekordnorth.co.za/i-love-my-city/',
-            'rekordnorth.co.za/?p=128157',
-            'rekordnorth.co.za/student-living/',
-            'rekordnorth.co.za/be-your-best/',
-            'rekordnorth.co.za/magalieskruin-centre/',
-            'rekordnorth.co.za/about-us',
-            '//biz.rekordnorth.co.za/',
-            '//www.guzzle.co.za/',
-            '//www.autodealer.co.za/',
-            '//iab.com/',
-            '//www.bestofpretoria.co.za/',
-            '//localnewsnetwork.co.za/',
-            '//www.caxton.co.za/',
-            '//facebook.com/',
-            '//twitter.com/',
-            '//southcoastherald.co.za/',
-            '//rekordcenturion.co.za/',
-            '//kemptonexpress.co.za/',
-            '//northglennews.co.za/',
-            '//albertonrecord.co.za/',
-            '//ladysmithgazette.co.za/',
-            '//witbanknews.co.za/',
-            '//roodepoortrecord.co.za/',
-            '//southerncourier.co.za/',
-            '//krugersdorpnews.co.za/',
-        )
+        allow=('https://rekordnorth.co.za/\d+/', ),
+        allow_domains=allowed_domains
     )
 
     rules = (
```